tests/modules/nf-core/ashlar/zero_uuid.nf

```diff
@@ -0,0 +1,32 @@
+nextflow.enable.dsl=2
+
+process ZERO_UUID {
+
+    input:
+    each path(file_in)
+
+    script:
+    """
+    TARG_PATHS_ARRAY=(\$(find ../.. -name "ashlar_output.ome.tif" -printf "%k %p\n" | sort -n | awk -F ' ' '{print \$2}'))
```

Comment:
Even though I think we want to take a totally different approach here, I want to point out a general Nextflow thing here: This relative path access is not generally OK! You can only rely on what you were given in the `input:` file args -- traversing the filesystem is not going to work when the files were staged from e.g. cloud storage and that other file isn't present in the local work directory.

Reply:
Oh, sure, thanks for pointing that out.  I was thinking that this directory structure would always be present, but that makes sense.

As you say, working from hard-coded hashes makes much more sense anyway.